content/applications/finance/accounting/others/analytic/analytic_accounting.rst

```diff
@@ -0,0 +1,122 @@
+- :guilabel:`Customer`: select the customer related to your project;
+- :guilabel:`Reference`: add a reference to make it easier to find the account when you are on your
+  bill;
+- :guilabel:`Plan`: an :guilabel:`Analytic Account` must be related to at least one
+  :guilabel:`Analytic plan`;
+- :guilabel:`Company`: select the company for which the :guilabel:`Analytic Account` will be used;
+- :guilabel:`Currency`: select the currency of the :guilabel:`Analytic Account`.
+
+Then, fill in your budget information.
+
+.. seealso::
+   :doc:`Financial budget <../adviser/budget>`.
+
+Analytic plans
+==============
+
+You can access the :guilabel:`Analytic Plans` by going to :menuselection:`Accounting -->
+Configuration --> Analytic Accounting: Analytic Plans`. Click :guilabel:`New` to create a new plan.
+
+.. image:: analytic_accounting/analytic_plans.png
+   :align: center
+   :alt: create an analytic plan
+
+The following information must be completed:
+
+- :guilabel:`Parent`: link your plan to another :guilabel:`Analytic Plan` to build a hierarchy
+  between your plans;
+- :guilabel:`Default Applicability`: :guilabel:`Optional`, :guilabel:`Mandatory`, or
+  :guilabel:`Unavailable`;
+- :guilabel:`Color`: select the color of the tag related to this specific plan;
+- :guilabel:`Company`: add the company to which the plan applies;
+- :guilabel:`Domain`: choose where your plan is applicable;
+- :guilabel:`Financial Accounts Prefix`: select the prefix of the account(s) to which this plan
+  should be applied;
+- :guilabel:`Product Category`: decide to which product category the plan applies;
+- :guilabel:`Applicability`: decide how your plan behaves in the widget when creating a new journal
+  entry:
+
+  - :guilabel:`Optional`: if selected, it is not mandatory to add the analytic plan in the widget;
+  - :guilabel:`Mandatory`: if selected, an orange bullet is visible in the widget next to the plan
+    until the analytic distribution is done (the bullet then turns to green); it is not possible to
+    confirm the entry if no analytic account is selected;
+  - :guilabel:`Unavailable`: if selected, the plan is not available in the widget.
+
+Two smart buttons are available in the top-right corner:
+
+   - :guilabel:`Subplans`: can be created to have a more complex analytic structure. Click the
+     :guilabel:`Subplans` smart button, and then :guilabel:`New` to add a subplan;
+   - :guilabel:`Analytic Accounts`: to reach the subplans related to the plan.
+
+.. note::
+   - The analytic widget is prefilled based on the :guilabel:`Applicability`, and the
+     :ref:`Analytic Distribution Models <analytic_distribution_models>`;
+   - Each :guilabel:`Analytic plan` must have at least one :guilabel:`Analytic Account`.
+
+Analytic distribution
+=====================
+
+When creating an invoice or a bill it is mandatory to add a plan in the
+:guilabel:`Analytic Distribution` column. This opens a **widget**, in which you can fill in the
```

Comment:
If an analytic account is related to your plan, then, yes. Otherwise not.
I already clarified in the note above that "Each analytic plan must have at least one analytic account."
I updated the phrasing to make it clearer :)